vendor-bin/doctrine_mongodb/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "autoload": {
```

Comment:
What an horrible thing to do 😱 !

More seriously, it's a bit tedious to do that and error prone, that's what `theofidry/composer-inheritance-plugin` is for so you shouldn't need this.